events/api.py

```diff
@@ -1483,18 +1482,54 @@ def get_has_regular_users(obj):
         return obj.regular_users.count() > 0
 
 
+class OrganizationUsersField(serializers.SlugRelatedField):
```

Comment:
As there is nothing that makes this field actually bound to multiple users (one can use it without `many=True` and that is nice) I would rather name it in singular form `OrganizationUserField`. Huge deal, I know.

Reply:
Fixed in https://github.com/City-of-Helsinki/linkedevents/pull/609/commits/25d0c77b46dbc5e272a40d47f18618f54b4a43ef